epicsdbbuilder/recordbase.py

```diff
@@ -158,6 +163,14 @@ def Print(self, output):
             print('    field(%s, %s%s)' % (k, padding, value), file = output)
         for alias in sorted(list(self.__aliases)):
             print('    alias("%s")' % alias, file = output)
+        for name, info in list(self.__infos):
+            if isinstance(info, str):
+                # A string value, double quote it
+                info = '"%s"' % info
```

Comment:
Check whether `recordbase.quote_string()` will do the right thing here, it would be better to use this if possible.

Reply:
Done